packages/ui-stencil/src/components/internal/orama-embed/orama-embed.tsx

```diff
@@ -0,0 +1,24 @@
+import { Component, h, Prop, State, Listen, Element, Event, type EventEmitter, Watch } from '@stencil/core'
+
+export type ModalStatus = {
+  open: boolean
+  id: HTMLElement
+}
```

Comment:
this is probably not needed, maybe you forgot to delete it

Reply:
done!